services/hanke-service/src/main/resources/db/changelog/changesets/create-organization-tables.yml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-organization-tables
+      comment: Create organization tables
+      author: Anne-Mari Silvast
+      changes:
+        - createTable:
+            tableName: organisaatio
+            columns:
+              - column: { name: id, type: int, autoIncrement: true, constraints: { primaryKey: true, nullable: false }}
+              - column:
+                  name: organisaatiotunnus
+                  type: varchar(15)
+                  constraints:
+                    unique: true
+              - column: { name: nimi, type: varchar(100) }
+              - column: { name: createdat, type: timestamp }
+              - column: { name: modifiedat, type: timestamp }
```

Comment:
Missing audit fields for creator (createdByUserId) and modifier (modifiedByUserId).

Reply:
This is not the table where user organisations are added. It will be its own table. This is just the static list that cannot be modified by users.

Reply:
But this will data eventually also be modified by user when there will be some kind of management possibility for Admin users.
Of course the columns could be added afterwards too. :)